### PR TITLE
EZP-30398: Cleanup code style for JavaScript

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30398
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

### CODE REVIEW

**Comment by tischsoic** [comment]
Fixes according to comments in last commit: https://github.com/ezsystems/ezplatform-admin-ui/pull/971/commits/8eb56751c4866270b9470206d2cb1154f6107ae0 - ping @sunpietro @dew326.

File: src/bundle/Resources/public/js/scripts/admin.contenttype.fieldtype.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
     const toggleFieldTypeView = (event) => {
         event.preventDefault();
 
         event.currentTarget.closest('.ez-card--fieldtype-container').classList.toggle('ez-card--collapsed');
     };
 
-    togglers.forEach(btn => btn.addEventListener('click', toggleFieldTypeView, false));
-})(window, window.document);
+    togglers.forEach((btn) => btn.addEventListener('click', toggleFieldTypeView, false));
+})(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.contenttype.fieldtype.js]: `window.document`?

File: src/bundle/Resources/public/js/scripts/admin.location.add.translation.js [hunk 2]
```diff
@@ -36,5 +37,5 @@
     };
 
     resetSelection();
-    [...doc.querySelectorAll(SELECTOR_LABEL)].forEach((input) => input.addEventListener('click', updateSelection, false));
+    doc.querySelectorAll(SELECTOR_LABEL).forEach((input) => input.addEventListener('click', updateSelection, false));
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.location.add.translation.js]: `window.document`?

File: src/bundle/Resources/public/js/scripts/admin.location.set.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-    const updateMainLocation = event => {
+(function(global, doc) {
+    const updateMainLocation = (event) => {
         doc.querySelector('#content_location_set_main_location').value = event.target.value;
         doc.querySelector('form[name="content_location_set_main"]').submit();
     };
 
-    [...doc.querySelectorAll('input[name="setMainLocation"]')].forEach(input => input.addEventListener('change', updateMainLocation, false));
+    doc.querySelectorAll('input[name="setMainLocation"]').forEach((input) => input.addEventListener('change', updateMainLocation, false));
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.location.set.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/admin.location.update.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
...
+    const updateMainLocation = (event) => {
         doc.querySelector('#content_main_location_update_location').value = event.target.value;
         doc.querySelector('form[name="content_main_location_update"]').submit();
     };
 
-    [...doc.querySelectorAll('input[name="updateMainLocation"]')].forEach(input => input.addEventListener('change', updateMainLocation, false));
+    doc.querySelectorAll('input[name="updateMainLocation"]').forEach((input) =>
+        input.addEventListener('change', updateMainLocation, false)
+    );
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.location.update.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/admin.preview.js [hunk 5]
```diff
@@ -12,14 +12,16 @@
...
 
         iframeWrapper.setAttribute('src', siteaccessPreviewUrl);
     };
-    [...doc.querySelectorAll(SELECTOR_BTN_ACTION)].forEach(btn => btn.addEventListener('click', changePreviewMode, false));
-    [...doc.querySelectorAll(SELECTOR_PREVIEW_SITEACCESS_SELECT)].forEach(select => select.addEventListener('change', changePreviewSiteaccess, false));
+    doc.querySelectorAll(SELECTOR_BTN_ACTION).forEach((btn) => btn.addEventListener('click', changePreviewMode, false));
+    doc.querySelectorAll(SELECTOR_PREVIEW_SITEACCESS_SELECT).forEach((select) =>
+        select.addEventListener('change', changePreviewSiteaccess, false)
+    );
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/admin.preview.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js [hunk 6]
```diff
@@ -1,13 +1,17 @@
...
-    }, false);
+    editButton.addEventListener(
+        'click',
+        () => {
+            languageRadioOption.checked = true;
+            languageRadioOption.dispatchEvent(new CustomEvent('change'));
+        },
+        false
+    );
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/sidebar/btn/contenttype.edit.js [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
     const editButton = doc.querySelector('.ez-btn--edit');
     const languageRadioOption = doc.querySelector('.ez-extra-actions--edit.ez-extra-actions--prevent-show [type="radio"]');
-    const editActions = doc.querySelector('.ez-extra-actions--edit')
-    const btns = [...editActions.querySelectorAll('.form-check [type="radio"]')];
+    const editActions = doc.querySelector('.ez-extra-actions--edit');
+    const btns = editActions.querySelectorAll('.form-check [type="radio"]');
     const changeHandler = () => {
         const form = doc.querySelector('.ez-extra-actions--edit form');
-        form.submit()
+        form.submit();
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/btn/contenttype.edit.js]: empty line after `const`

File: src/bundle/Resources/public/js/scripts/sidebar/btn/location.create.js [hunk 8]
```diff
@@ -1,7 +1,7 @@
...
-    const btns = [...createActions.querySelectorAll('.form-check [type="radio"]')];
+(function(global, doc) {
+    const createActions = doc.querySelector('.ez-extra-actions--create');
+    const btns = createActions.querySelectorAll('.form-check [type="radio"]');
     const form = createActions.querySelector('form');
 
-    btns.forEach(btn => btn.addEventListener('change', () => form.submit(), false));
-})();
+    btns.forEach((btn) => btn.addEventListener('change', () => form.submit(), false));
+})(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/btn/location.create.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/sidebar/btn/user.edit.js [hunk 9]
```diff
@@ -7,8 +7,12 @@
...
-    }, false);
+    editButton.addEventListener(
+        'click',
+        () => {
+            languageRadioOption.checked = true;
+            languageRadioOption.dispatchEvent(new CustomEvent('change'));
+        },
+        false
+    );
 })(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/btn/user.edit.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js [hunk 10]
```diff
@@ -1,53 +1,57 @@
...
 
-            actions.style.opacity = 1;
-        }, false);
+                actions.style.opacity = 1;
+            },
+            false
+        );
     });
-})(window, window.document);
+})(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js]: `window.document`

File: src/bundle/Resources/public/js/scripts/sidebar/instant.filter.js [hunk 11]
```diff
@@ -27,15 +27,21 @@
...
+        );
 
         filterInput.addEventListener('change', filterItems.bind(filter, itemsMap, groups), false);
         filterInput.addEventListener('blur', filterItems.bind(filter, itemsMap, groups), false);
         filterInput.addEventListener('keyup', filterItems.bind(filter, itemsMap, groups), false);
     };
 
     filters.forEach(initFilter);
-})();
+})(window, document);
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/sidebar/instant.filter.js]: `window.document`

**Pull request merged**